### ISSUE TITLE
Module replaces groups imported to terraform state

### ISSUE THREAD

**Issue by ploth** [terraform-google-modules/terraform-google-group#67]
### TL;DR

None of the possible values for `initial_group_config` match groups initially created via the admin console (UI). This leads to a replacement of the group.

### Expected behavior

* I can use this module with groups created via admin console.
* I can specify the group resource identically to already existing groups so that I'm not forced to replace groups I gonna manage via terraform
* I can set `initial_group_config` to a value which doesn't lead to group replacement.

### Observed behavior

I'm about to import google groups to my terraform state. The groups were created manually via the admin console. 

The parameter `initial_group_config` defaults to "EMPTY" if left undefined in the terraform resource definition. **"EMPTY" or any other possible value (`INITIAL_GROUP_CONFIG_UNSPECIFIED`, `WITH_INITIAL_OWNER `) forces the group to be recreated/replaced.**

Without replacing the groups its not possible to use this module for groups initially created outside of terraform.

#### Temporary workaround

I forked this module and added `initial_group_config` to `lifecycle.ignore_changes`.

#### Possible Fix

I think it should be fixed by making this [line](https://github.com/terraform-google-modules/terraform-google-group/blob/main/main.tf#L42) dynamic depending on whether `INITIAL_GROUP_CONFIG_UNSPECIFIED` is set.

### Terraform Configuration

```hcl
1. Create a google group `foobar` with email `foo@bar.de` via admin.google.com.
2. Add `email@address.de` as owner.
3. Import (`terraform import`) group to `module.foobar.google_cloud_identity_group.group` and membership to `module.foobar.google_cloud_identity_group_membership.owners["email@address.de"]
4. Apply terraform


module "foobar" {
  source      = "terraform-google-modules/group/google"
  version     = "0.6.1"
  customer_id = "foo"

  id           = "foo@bar.de"
  display_name = "foobar"
  types        = ["default"]
  owners = [
    "email@address.de",
  ]
}
```
```


### Terraform Version

```sh
Terraform v1.5.7
on darwin_arm64
+ provider registry.terraform.io/hashicorp/google v5.26.0
```
```


### Additional information

_No response_
